src/System.Security.Cryptography.Pkcs/src/MatchingRefApiCompatBaseline.netstandard.txt

```diff
@@ -1,3 +1,6 @@
 # These members don't belong in netstandard reference because of type unification with netfx (the members are new in net472).
 MembersMustExist : Member 'System.Security.Cryptography.Pkcs.SignerInfo.GetSignature()' does not exist in the reference but it does exist in the implementation.
 MembersMustExist : Member 'System.Security.Cryptography.Pkcs.SignerInfo.SignatureAlgorithm.get()' does not exist in the reference but it does exist in the implementation.
+
+MembersMustExist : Member 'System.Security.Cryptography.CryptographicAttributeObjectCollection.IsSynchronized.get()' does not exist in the reference but it does exist in the implementation.
+MembersMustExist : Member 'System.Security.Cryptography.CryptographicAttributeObjectCollection.SyncRoot.get()' does not exist in the reference but it does exist in the implementation.
```

Comment:
This seems off.  This is just comparing the implementation to the netstandard build of the ref, but you changed the ref.

So it feels like changes to this file shouldn't be necessary...